cou/zaza_utils/openstack.py

```diff
@@ -0,0 +1,155 @@
+import logging
```

Comment:
If we are coping something from zaza, I think that best practice would be do in in separate commit and use commit hash from which it was copied.

Reply:
or use a submodule if you can't simply import the package